### PR TITLE
Merge the Release Branch Back into Main

### PR DESCRIPTION
**Pull request by dbernstein**
Standard practice as part of the release process.
# Interested parties
Tag (@ mention) interested parties or, if unsure, @fcrepo/committers

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -22,7 +22,7 @@
       (for scm, site-distribution, etc -->
     <project_name>fcrepo</project_name>
     <project_organization>fcrepo</project_organization>
-    <fcrepo-storage-ocfl.version>6.3.0-SNAPSHOT</fcrepo-storage-ocfl.version>
+    <fcrepo-storage-ocfl.version>6.3.0</fcrepo-storage-ocfl.version>
```
Comment (whikloj) [inline pom.xml:25]: Shouldn't this be `6.4.0-SNAPSHOT` now?

Comment (whikloj) [reply pom.xml:25]: @dbernstein when you have a moment (so we don't forget to get these changes back in main), can you check this? I feel like it should be altered.

Comment (dbernstein) [reply pom.xml:25]: sure - I'll fix that.

**Review by whikloj — APPROVED** [review]
👍

**Pull request merged**